cql/pom.xml

```diff
@@ -94,7 +94,11 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <scope>provided</scope>
```

Comment:
Shouldn't this be `test`? I don't think it should come be used outside of test code, or be available.